packages/discord.js/src/structures/SelectMenuInteraction.js

```diff
@@ -18,53 +18,53 @@ class SelectMenuInteraction extends MessageComponentInteraction {
 
     const { members, users, roles, channels } = data.data.resolved || {};
 
-    if (members) {
+    if (this.isUserSelectMenu() || this.isMentionableSelectMenu()) {
```

Comment:
i said dont do it

Reply:
and i moved collections to deffrent classes

Reply:
i said it works, and they really are pretty identical

Reply:
> i said it works, and they really are pretty identical

typeguarding by .componentType not

Reply:
and now its just simpler

Reply:
no that wouldn't work, isn't that's why we added the typeguard functions?